ExtLibs/ArduPilot/Mavlink/MAVLinkInterface.cs

```diff
@@ -307,7 +307,7 @@ public MAVState MAV
             set { MAVlist[sysidcurrent, compidcurrent] = value; }
         }
 
-        public double CONNECT_TIMEOUT_SECONDS = Settings.Instance.GetDouble("CONNECT_TIMEOUT_SECONDS", 30.0);
+        public double CONNECT_TIMEOUT_SECONDS = Settings.Instance.GetDouble("CONNECT_TIMEOUT_SECONDS", 9999.0);

```

Comment:
why the long timeout?

Reply:
i agree, default should stay the same, you can override via the setting

Reply:
the long timeout is due to the client in  question needing to be able to "connect" to their rfd900 radio while the vehicle at the other end is out-of-range, and have the vehicle auto-connect immediately when it comes in-range, it should probably be both configurable and a separate PR, but i haven't got the yet.   marking this PR as WIP/draft.